analytics/trend_analyzer.py

```
@@ -0,0 +1,631 @@
+        return dict(daily_stats)
+    
+    def _aggregate_weekly_stats(self, daily_stats: Dict[str, Any]) -> Dict[str, Any]:
+        """週次統計を集計"""
+        weekly_stats = defaultdict(lambda: {
+            'conversations': 0,
+            'analyses': 0,
+            'avg_scores': defaultdict(list),
+            'days_active': 0
+        })
+        
+        for date_str, stats in daily_stats.items():
```

Ruff (0.12.2)

Import `stats` from line 9 shadowed by loop variable

(F402)